RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -185,8 +276,19 @@ private string CreateBaseWorkingDirectory()
             return baseDirectory;
         }
 
-        private IEnumerable<DiagnosticAnalyzer> GetAnalyzers(string packageRootDir, string additionalSearchFolder, string language)
+        private bool HasAnalyzers(IPackage package, string language)
+        {
+            return GetAnalyzers(package, language).Any();
```

Comment:
The log output still shows assemblies with the list of any the located analysers being shown twice. Is this deliberate?